### PR TITLE
create cta response component

### PR DESCRIPTION
**Pull request by afolasope**

### CODE REVIEW

**Review by jdamilola — APPROVED** [review]

**Review by AfeezGL — APPROVED** [review]

File: src/surveys/components/CTAResponse.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <div className={'border'}>
+      <Header
+        title={question?.label}
+        description={question?.description}
+        color={theme?.question ? theme?.question : '#050505'}
+      />
+
+      {(question?.settings as CTASettings).type === CTAType.CLOSE && (
+        <div>
+          <Button
```
Comment (jdamilola) [inline src/surveys/components/CTAResponse.tsx]: @afolasope, why do we have 3 buttons that all look the same?

**Review by jdamilola — CHANGES_REQUESTED** [review]

File: src/surveys/components/CTAResponse.tsx (hunk 1 above)
Comment (jdamilola) [reply src/surveys/components/CTAResponse.tsx]: @afolasope I feel we should just return `null` if type is `hidden`, then render the same button for other types.

Comment (afolasope) [reply src/surveys/components/CTAResponse.tsx]: I have fixed it

**Review by jdamilola — APPROVED** [review]

**Pull request merged**